src/utility/index.jsx

```diff
@@ -592,3 +592,18 @@ export const wilsonScore = (up, down) => {
     1 + (z * z / n)
   );
 };
+
+export const commonConsumableItems = [
+  'Town Portal Scroll',
```

Comment:
actually I mean the internal item name, like `tpscroll`, since those are unlikely to change.

Reply:
items already had a qual property. I'm so blind :)